app/questionnaire/questionnaire_schema.py

```diff
@@ -36,6 +36,7 @@
 DependencyDictType: TypeAlias = dict[str, OrderedSet[str]]
 
 TRANSFORMS_REQUIRING_ROUTING_PATH = {"first_non_empty_item"}
+TRANSFORMS_REQUIRING_UNRESOLVED_ARGUMENTS = ["format_currency"]
```

Comment:
Minor for consistency, this and the arg name should match, both should either be raw arg or unresolved arg.

Reply:
Updated 👍